src/Symfony/Component/Form/Extension/Core/DataMapper/FormDataToObjectConverterInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Extension\Core\DataMapper;
+
+/**
+ * @author Maxime Steinhausser <maxime.steinhausser@gmail.com>
+ */
+interface FormDataToObjectConverterInterface
+{
+    /**
+     * Convert the form data into an object.
```

Comment:
Converts.

Reply:
This interface does not extend any other interface.